components/SetUpStaking/StakingStepper.jsx

```diff
@@ -0,0 +1,165 @@
+import { useState, useEffect } from 'react'
+import { useSelector } from 'react-redux';
+import { Flex, Steps, Typography, Button, Skeleton } from 'antd';
+import Link from 'next/link';
+import { notifyError } from '@autonolas/frontend-library';
+import { truncateAddress, formatToEth, getAddressFromBytes32 } from 'common-util/functions';
+import ConnectTwitterModal from '../ConnectTwitter/Modal';
+import { useTotalBond, checkAndApproveToken, createAndStake } from './requests';
+
+const { Paragraph, Text } = Typography;
+
+const XDAI_FOR_GAS = 2;
```

Comment:
Please leave a comment here:
1 wei for the service registration activation, and 1 wei for agent instance bond, which is always equal to 1 for now.

Reply:
Probably need to rename into:
`XDAI_FOR_DEPOSIT`

Reply:
Ahhhh ok that's for operating it, sorry )

Reply:
I'm actually unsure if it'll be enough, will leave a comment to verify that later

Reply:
Contributors don't do any on-chain work :) it's all contribute service that is sweating for them on-chain. They just need gas to call claim and other actions from time to time. So 2 XDai is more than enough for the first operating balance.